src/rhelocator/api/server.py

```diff
@@ -47,6 +48,10 @@ def create_app(file_path: str) -> Flask:
         aws_blueprint(image_data["images"]["aws"]), url_prefix="/api"
     )
 
+    app.register_blueprint(
+        health_blueprint(image_data["images"]), url_prefix="/api"
```

Comment:
Currently the api endpoint is reachable under `/api/health` does this make sense? Or should we use just `/health` instead?